modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -271,14 +276,36 @@ public void createBucket(BucketSettings bucketSetting, UserSettings userSettings
         }
     }
 
+    private void createFullTextIndex() {
+        String urlBaseFts = String.format("http://%s:%s", getContainerIpAddress(), getMappedPort(FTS));
+        String fullTextIndexURL = urlBaseFts + "/api/index/" + ftsName;
+        StringBuilder fullTextIndex = new StringBuilder();
+        try {
+            Path path = Paths.get(CouchbaseContainer.class.getClassLoader()
+                .getResource(ftsName + ".json").toURI());
```

Comment:
Should we really append the `.json` file extension here? Maybe more intuitive for the user to just specify the file name.